tools/generate_formula_api.py

```diff
@@ -43,16 +43,15 @@ def iter_subclasses(cls, _seen=None, template_classes=[]):
 def write_formula_api(directory):
     template_classes = ['DiscreteModel', 'BinaryModel', 'MultinomialModel',
                         'OrderedModel', 'CountModel',
-                      'LikelihoodModel', 'GenericLikelihoodModel',
-                      'TimeSeriesModel',
-                      # this class should really be deleted
-                      'ARIMAProcess',
-                      # these need some more work, so don't expose them
-                      'ARIMA', 'VAR', 'SVAR', 'AR', 'NBin', 'NbReg', 'ARMA',
-                      ]
```

Comment:
some other tools recommend this to minimize line changes when adding additional terms

Reply:
I'll see if I can find a way to make both tools happy.